api/composer.json

```diff
@@ -52,6 +52,7 @@
         "friendsofphp/php-cs-fixer": "3.14.3",
         "hautelook/alice-bundle": "2.11.0",
         "justinrainbow/json-schema": "5.2.12",
+        "doctrine/annotations": "2.0.0 as 1.14.2",
```

Comment:
Didn't know this is possible. Looks a bit hacky to me.

@BacLuc:
Which packages depend on doctrine/annotations v1? And how do we ensure, these packages don't fail due to some BC in v2?

Reply:
Its for example gedmo, which is updated in https://github.com/ecamp/ecamp3/pull/3244 which allows doctrine/annotations v2.0.
I am at it

Reply:
Yes, after #3244, this pr works without hack